### PR TITLE
[v7r1] Bugs fixes.

### PR DESCRIPTION
**Pull request by TaykYoku**
BEGINRELEASENOTES
This PR will fix some bugs

*Core:
FIX: test if proposalTuple before read clientVersion in /DISET/private/Service

*FrameworkSystem:
FIX: Add None type for vomsAttribute in ProxyManagerHandler

*Resources:
NEW: Add test for None proxyProvider in ProxyProviderFactory
FIX: Fix bugs and change test for DIRACCAProxyProvider

ENDRELEASENOTES

### CODE REVIEW

File: Resources/ProxyProvider/ProxyProviderFactory.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def getProxyProvider(self, proxyProvider):
     ppDict = result['Value']
     ppDict['ProviderName'] = proxyProvider
     ppType = ppDict.get('ProviderType')
+    if not ppType:
+      S_ERROR('Cannot found describe for %s' % proxyProvider)
```
Comment (andresailer) [inline Resources/ProxyProvider/ProxyProviderFactory.py]:
```suggestion
      return S_ERROR('Cannot find information for ProxyProvider %s' % proxyProvider)
```
